js/ui/grid_core/ui.grid_core.column_fixing.js

```diff
@@ -637,29 +637,41 @@ const RowsViewFixedColumnsExtender = extend({}, baseFixedColumns, {
         }
     },
 
+    _getScrollDelay: function() {
+        let scrollDelay = browser.mozilla ? 60 : 0;
+        const hasResizeTimeout = this.getController('resizing')?.hasResizeTimeout();
+        const updateTimeout = this.option('scrolling.updateTimeout');
+
+        if(hasResizeTimeout) {
+            scrollDelay = updateTimeout;
+        }
+
+        return scrollDelay;
+    },
```

Comment:
Let's make this function shorter:

```suggestion
    _getScrollDelay: function() {
        const hasResizeTimeout = this.getController('resizing')?.hasResizeTimeout();
        
        if (hasResizeTimeout) {
            return this.option('scrolling.updateTimeout');
        }
        
        return browser.mozilla ? 60 : 0;
    },
```